src/test/java/ibm/jceplus/junit/base/BaseTestAESGCM_IntIV.java

```diff
@@ -66,46 +63,11 @@ protected void setUp() throws Exception {
 
         key = aesKeyGen.generateKey();
 
```

Comment:
Why remove classGCMParameterSpec and ctorGCMParameterSpec and the code setting those values. Are these fields unused?

Reply:
These fields were used because the code accounted for Java <8 versions and needed to instantiate the proper class. But for our purposes, we know exactly what class and constructor we want and can use them directly.